dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+SET sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
+
+-- uc_dolphin_T_t_ds_command_R_test_flag
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_command_R_test_flag;
+delimiter d//
+CREATE PROCEDURE uc_dolphin_T_t_ds_command_R_test_flag()
+BEGIN
+       IF NOT EXISTS (SELECT 1 FROM information_schema.COLUMNS
+           WHERE TABLE_NAME='t_ds_command'
+           AND TABLE_SCHEMA=(SELECT DATABASE())
+           AND COLUMN_NAME ='test_flag')
+   THEN
+ALTER TABLE t_ds_command ADD `test_flag` tinyint(4) DEFAULT null COMMENT 'test flag：0 normal, 1 test run';
+END IF;
+END;
+
+d//
+
+delimiter ;
+CALL uc_dolphin_T_t_ds_command_R_test_flag;
+DROP PROCEDURE uc_dolphin_T_t_ds_command_R_test_flag;
+
+-- uc_dolphin_T_t_ds_error_command_R_test_flag
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_error_command_R_test_flag;
+delimiter d//
+CREATE PROCEDURE uc_dolphin_T_t_ds_error_command_R_test_flag()
+BEGIN
+       IF NOT EXISTS (SELECT 1 FROM information_schema.COLUMNS
+           WHERE TABLE_NAME='t_ds_error_command'
+           AND TABLE_SCHEMA=(SELECT DATABASE())
+           AND COLUMN_NAME ='test_flag')
+   THEN
+ALTER TABLE t_ds_error_command ADD `test_flag` tinyint(4) DEFAULT null COMMENT 'test flag：0 normal, 1 test run';
+END IF;
+END;
+
+d//
+
+delimiter ;
+CALL uc_dolphin_T_t_ds_error_command_R_test_flag;
+DROP PROCEDURE uc_dolphin_T_t_ds_error_command_R_test_flag;
+
+-- uc_dolphin_T_t_ds_datasource_R_test_flag_bind_test_id
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_datasource_R_test_flag_bind_test_id;
+delimiter d//
+CREATE PROCEDURE uc_dolphin_T_t_ds_datasource_R_test_flag_bind_test_id()
+BEGIN
+       IF NOT EXISTS (SELECT 1 FROM information_schema.COLUMNS
+           WHERE TABLE_NAME='t_ds_datasource'
+           AND TABLE_SCHEMA=(SELECT DATABASE())
+           AND COLUMN_NAME ='test_flag'
```

Comment:
sql is wrong, COLUMN_NAME ='test_flag' AND COLUMN_NAME ='bind_test_id'

Reply:
sure, is done, This will be submitted after the next comment is discussed.